docusaurus/docs/user-docs/settings/review-workflows.md

```diff
@@ -5,18 +5,40 @@ description: Managing your review workflows and stages in Strapi.
 
 # Managing Review Workflows <EnterpriseBadge />
 
-The Review Workflows feature allows you to create and manage any desired review stages for your content, enabling your team to collaborate in the content creation flow from draft to publication. Before being available in the [Content Manager](/user-docs/content-manager/reviewing-content) and [Content-Types Builder](/user-docs/content-type-builder), review workflows should be configured from ![Settings icon](/img/assets/icons/settings.svg) *Settings > Global settings > Review Workflows*.
+The Review Workflows feature allows you to create and manage different review workflows for different content types. Each workflow can consist of any review stages for your content, enabling your team to collaborate in the content creation flow from draft to publication. Before being available in the [Content Manager](/user-docs/content-manager/reviewing-content) and [Content-Types Builder](/user-docs/content-type-builder), review workflows should be configured from ![Settings icon](/img/assets/icons/settings.svg) *Settings > Global settings > Review Workflows*.
+
+- should we explain a use case for multiple workflows?
 
 :::note
 - The Review workflows settings are only available to users with the Super Admin role by default. Other roles must be granted the **Review workflows** permission. See [Users, Roles, & Permissions](/user-docs/users-roles-permissions) for more information.
 - Review Workflows are only available with the Enterprise Edition. If for any reason your project was downgraded to the Community Edition, all review workflows data would not be deleted but kept until the project goes back to Enterprise.
 :::
 
-![Review Workflows Settings](/img/assets/review-workflows/review-workflows.png)
+## Listing all workflows
+
+![Review Workflows list view](/img/assets/review-workflows/review-workflows.png)
+
+- lists all workflows
+- allows creating new workflows
+- last workflow can not be deleted
+
+## Creating or editing a workflow
+
+![Review Workflows create and edit views](/img/assets/review-workflows/review-workflows.png)
+
+- users can create/ edit workflows
+- enable review-workflow on a content-type; if the content-type was already selected for another workflow, it will be re-assigned (as `:::note` ?)
+
+Questions:
+
+- should we mention license limits as `:::note`? (20 workflows / 200 stages per workflow)
+- should we mention hard limits in case an offline license is used? (200 workflows / 200 stages per workflow)
```

Comment:
@derrickmehaffy fyi: in case you want to add more information about limits.

https://www.notion.so/Multiple-workflows-Pricing-decision-84dd9e30f1024992aef36c613102a05c

Reply:
Yeah we don't mention the license specifics in the docs but thanks for the link @gu-stav I'll see next week if we want to add something to a support article.